dockerfiles/api/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:alpine
+WORKDIR /opt
+COPY package.json package-lock.json dist/apps/api ./
```

Comment:
you're copying the whole repo with ./ ?

Reply:
No, I'm copying the preceding files to the current working directory on the docker image.